index.html

```diff
@@ -148,17 +257,19 @@ <h2 class="content-text">About
                     <li class="separator"></li>
                   </ul>
             </div>
-            
+            </div>
+        </div>
         </div>
         <footer> 
             <section class="contact-me">
-                <div class="content">
+                <div class="contact-me-1">
```

Comment:
Kindly also make the contact me section resemble the Figma design 🤗
- _Please remove the extra space around it_
- _Fix the spacing between elements and Align them_
- _Make the form and text centered as shown in Figma_

Your design:

![image](https://user-images.githubusercontent.com/73714615/188266109-60a8c46e-d34a-41eb-ace3-bcb707025420.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/188266096-895695ac-5b9d-4b6c-b0c9-bee719871b67.png)

Reply:
Good job following the advice of the previous reviewer